test/integration/default/inspec.yml

```diff
@@ -15,12 +15,12 @@
 name: cloud-pubsub
 title: Google Cloud Pub/Sub
 version: 0.1.0
-inspec_version: '~> 2.3.24'
+inspec_version: '~> 3.0.64'
```

Comment:
This argument should just be dropped; Kitchen-Terraform manages the version of InSpec used.